site/committers.html

```diff
@@ -613,8 +613,8 @@ <h4>Merge script</h4>
 <p>Also, please remember to set Assignee on JIRAs where applicable when they are resolved. The script 
 can do this automatically in most cases. However where the contributor is not yet a part of the
 Contributors group for the Spark project in ASF JIRA, it won&#8217;t work until they are added. Ask
-an admin to add the person to Contributors at 
-https://issues.apache.org/jira/plugins/servlet/project-config/SPARK/roles .</p>
+an admin to add the person to Contributors at
+<a href="https://issues.apache.org/jira/plugins/servlet/project-config/SPARK/roles">https://issues.apache.org/jira/plugins/servlet/project-config/SPARK/roles</a> .</p>
```

Comment:
We can remove this outdated sentences completely because of https://github.com/apache/spark/pull/42653
> However where the contributor is not yet a part of the
Contributors group for the Spark project in ASF JIRA, it won&#8217;t work until they are added. Ask ...


cc @yaooqinn who is the main author of that improvement.

Reply:
Done.

Reply:
+1